README.adoc

```diff
@@ -0,0 +1,227 @@
+ARTIFACT_DIR                List of directories from which archives will be copied into the deployment folder. If unspecified, all archives in /target will be copied.                                                                                                                                              
+----
+
+== Create an Application Image
+
+[source,bash]
+----
+$ oc process eap-s2i-build \
+  -p APPLICATION_IMAGE=my-app \
+  \
+  -p EAP_IMAGE=jboss-eap-xp1-openjdk11-openshift:1.0 \
+  -p EAP_RUNTIME_IMAGE=jboss-eap-xp1-openjdk11-runtime-openshift:1.0 \
+  -p EAP_IMAGESTREAM_NAMESPACE=$(oc project -q) \
+  \
+  -p SOURCE_REPOSITORY_URL=https://github.com/jboss-developer/jboss-eap-quickstarts.git \
+  -p SOURCE_REPOSITORY_REF=xp-1.0.x\
+  -p CONTEXT_DIR=microprofile-config | oc create -f -
+
+imagestream.image.openshift.io/my-app created
+imagestream.image.openshift.io/my-app-build-artifacts created
+buildconfig.build.openshift.io/my-app-build-artifacts created
+buildconfig.build.openshift.io/my-app created
+----
+
+You can see all the resources created by this template by filtering with the label `app.kubernetes.io/name=<application image>`. For example:
+
+[source,bash]
+----
+$ oc get all -l "app.kubernetes.io/name=my-app"
+NAME                                                    TYPE     FROM           LATEST
+buildconfig.build.openshift.io/my-app                   Docker   Dockerfile     1
+buildconfig.build.openshift.io/my-app-build-artifacts   Source   Git@xp-1.0.x   2
+
+NAME                                                TYPE     FROM           STATUS    STARTED   DURATION
+build.build.openshift.io/my-app-build-artifacts-2   Source   Git@xp-1.0.x   Pending
+
+NAME                                                    IMAGE REPOSITORY                                                                      TAGS   UPDATED
+imagestream.image.openshift.io/my-app                   default-route-openshift-image-registry.apps-crc.testing/demo/my-app
+imagestream.image.openshift.io/my-app-build-artifacts   default-route-openshift-image-registry.apps-crc.testing/demo/my-app-build-artifacts
+----
+
+First, the build-artifacts image will be built using the EAP S2I Builder image:
+
+[source,bash]
+----
+$ oc logs -f bc/my-app-build-artifacts
+Cloning "https://github.com/jboss-developer/jboss-eap-quickstarts.git" ...
+...
+Successfully pushed image-registry.openshift-image-registry.svc:5000/demo/my-app-build-artifacts@sha256:01dc775bb76e00cbd8ccd31d4887e274ebcd6d2279aad39bc98a4fa749da974d
+Push successful
+----
+
+Then the actual application image will be built by using the EAP S2I Runtime image so that it contains only the runtime required to run the application:
+
+[source,bash]
+----
+$ oc logs -f bc/my-app-build-artifacts
```

Comment:
Should this be the same command as in previous step and not the `oc logs -f bc/my-app` instead?

Reply:
fixed, thanks